### ISSUE TITLE
A07_SECA_Rental_Fee

### ISSUE THREAD

**Issue by chrisasma103** [NaomiMod/Rent-A-Hero-No1#220]
The rental fee request from the bank is strangely worded. Below, it's "SECA has requested rental fee to pay 200 gold for your rental fee." I don't know what the control code's are, but I would reword it as "SECA has requested a rental fee of {RENTAL_FEE}."
![Rent A Hero No 1-221119-161042](https://user-images.githubusercontent.com/41581802/205997027-65c111b2-cfd1-4784-85fa-868a902ac371.png)

**Comment by ToddlerPichu** [comment]
The control code is `[UNK_VALUE6]`, but don't worry about typing that out in reports. I agree that this is worded strangely and will revise it... there is actually a bit more going on in this text compared to what it looks like with control codes, so revision may be a bit different but should be possible. Can you tell me what action was taken to get this text so I can revise and check it in game to make sure it functions? (who you talked to, did you upgrade something, etc. stuff like that)

**Comment by ToddlerPichu** [comment]
Thank you, that information will help me find the text in game... I will revise and test it when I do.

**Comment by ToddlerPichu** [comment]
Revised and tested in game... all should be good now.
